src/main/java/com/CezaryZal/manager/builder/TokenBuilder.java

```diff
@@ -0,0 +1,32 @@
+package com.CezaryZal.manager.builder;
+
+import com.CezaryZal.config.SecurityConstants;
+import com.CezaryZal.entity.User;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+
+@Service
+public class TokenBuilder {
+
+    public String buildTokenByUser(User foundUser){
+        long currentTimeMillis = System.currentTimeMillis();
```

Comment:
czy to na pewno jest prawidłowo? czy currentTimeMillis nie dostanie wartości w momencie inicjowania beana i na zawsze będzie miał te samą wartość?

Czy token wygenerowany po czasie życia aplikacji dłuższym niż czas ważności tokenu będzie ważny?

Reply:
Ooo to jest ważna uwaga.
Nie sprawdzałem dokładnie długości czasu tokenu więc patrząc na kod masz rację. 
Jest to czas inicjowania, wrzucę do buildera.